### PR TITLE
[NUI] Picker style update

### PR DESCRIPTION
**Pull request by lwc0917**
### Description of Change ###
Picker style modify

### API Changes ###
none

### CODE REVIEW

File: src/Tizen.NUI.Components/Theme/DefaultTheme.cs [hunk 1]
```diff
@@ -108,10 +108,15 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
 
                 // Picker
                 (new ExternalThemeKeyList(typeof(Picker), typeof(PickerStyle)))
+                    .Add<Size>("/Size", (ViewStyle style, Size value) => ((ViewStyle)style).Size = value)
+                    .AddSelector<float?>("/ItemTextLabel/PixelSize", SetButtonTextPixelSize, ControlState.Selected)
```
Comment (jaehyun0cho) [inline src/Tizen.NUI.Components/Theme/DefaultTheme.cs:112]: Please change this as follows.

.AddSelector<float?>("/ItemTextLabel/PixelSize", (ViewStyle style, Selector<float?> value) => ((PickerStyle)style).ItemTextLabel.PixelSize = value, ControlState.Selected)

File: src/Tizen.NUI.Components/Theme/DefaultTheme.cs [hunk 2]
```diff
@@ -108,10 +108,15 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
...
+                    .Add<Size>("/ItemTextLabel/Size", (ViewStyle style, Size value) => ((ButtonStyle)style).Text.Size = value)
```
Comment (jaehyun0cho) [inline src/Tizen.NUI.Components/Theme/DefaultTheme.cs:113]: Please change this as follows.

.AddSelector<Color>("/ItemTextLabel/TextColor", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).ItemTextLabel.TextColor = value, ControlState.Selected)

File: src/Tizen.NUI.Components/Theme/DefaultTheme.cs (hunk 1 above)
Comment (lwc0917) [reply src/Tizen.NUI.Components/Theme/DefaultTheme.cs]: Done

File: src/Tizen.NUI.Components/Theme/DefaultTheme.cs (hunk 2 above)
Comment (lwc0917) [reply src/Tizen.NUI.Components/Theme/DefaultTheme.cs:113]: Done

**Comment by lwc0917** [comment]
@jaehyun0cho  Thank you.

File: src/Tizen.NUI.Components/Theme/DefaultTheme.cs [hunk 3]
```diff
@@ -108,10 +108,15 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
 
                 // Picker
                 (new ExternalThemeKeyList(typeof(Picker), typeof(PickerStyle)))
-                    .AddSelector("/ItemTextLabel/Background", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).ItemTextLabel.BackgroundColor = value)
-                    .AddSelector("/ItemTextLabel/TextColor", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).ItemTextLabel.TextColor = value)
-                    .AddSelector("/ItemTextLabel/PixelSize", (ViewStyle style, Selector<float?> value) => ((PickerStyle)style).ItemTextLabel.PixelSize = value)
-                    .AddSelector("/Divider/Background", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).Divider.BackgroundColor = value),
+                    .Add<Size>("/Size", (ViewStyle style, Size value) => ((ViewStyle)style).Size = value)
+                    .AddSelector<float?>("/ItemTextLabel/PixelSize", (ViewStyle style, Selector<float?> value) => ((PickerStyle)style).ItemTextLabel.PixelSize = value, ControlState.Selected)
+                    .Add<Size>("/ItemTextLabel/Size", (ViewStyle style, Size value) => ((ButtonStyle)style).Text.Size = value)
```
Comment (jaehyun0cho) [inline src/Tizen.NUI.Components/Theme/DefaultTheme.cs:113]: Please change this as follows.

((ButtonStyle)style).Text.Size = value)         ->        ((PickerStyle)style).ItemTextLabel.Size = value)

File: src/Tizen.NUI.Components/Theme/DefaultTheme.cs [hunk 4]
```diff
@@ -108,10 +108,15 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
...
+                    .AddSelector<Color>("/ItemTextLabel/TextColor", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).ItemTextLabel.BackgroundColor = value, ControlState.Selected)
```
Comment (jaehyun0cho) [inline src/Tizen.NUI.Components/Theme/DefaultTheme.cs:114]: Please change this as follows.

((PickerStyle)style).ItemTextLabel.BackgroundColor = value, ControlState.Selected)            ->          ((PickerStyle)style).ItemTextLabel.TextColor = value, ControlState.Selected)

File: src/Tizen.NUI.Components/Theme/DefaultTheme.cs [hunk 5]
```diff
@@ -108,10 +108,15 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
...
+                    .AddSelector("/ItemTextLabel/Background", (ViewStyle style, Selector<Color> value) => ((PickerStyle)style).ItemTextLabel.BackgroundColor = value, ControlState.Selected)
```
Comment (jaehyun0cho) [inline src/Tizen.NUI.Components/Theme/DefaultTheme.cs:115]: Please change this as follows.

 .AddSelector("/ItemTextLabel/Background"             ->              .AddSelector("/ItemTextLabel/BackgroundColor"

File: src/Tizen.NUI.Components/Theme/DefaultTheme.cs (hunk 3 above)
Comment (lwc0917) [reply src/Tizen.NUI.Components/Theme/DefaultTheme.cs]: Done

File: src/Tizen.NUI.Components/Theme/DefaultTheme.cs (hunk 4 above)
Comment (lwc0917) [reply src/Tizen.NUI.Components/Theme/DefaultTheme.cs]: Done

File: src/Tizen.NUI.Components/Theme/DefaultTheme.cs (hunk 5 above)
Comment (lwc0917) [reply src/Tizen.NUI.Components/Theme/DefaultTheme.cs]: Done

**Comment by lwc0917** [comment]
@jaehyun0cho  Updated PR. Thanks.

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Comment by jaehyun0cho** [comment]
@lwc0917 Could you please rebase this patch to remove merge commit?

**Pull request merged**